eng/globalconfigs/NonShipping.globalconfig

```diff
@@ -0,0 +1,6 @@
+is_global = true
+
+### Configuration for RSxxxx rules ###
+
+# Do not call 'GetTestAccessor()'
+dotnet_diagnostic.RS0043.severity = none
```

Comment:
I see that NonShipping ruleset had lot more entries. Were they all redundant/stale ones?

Reply:
The way I went for this was to start with empty `globalconfig`s and add suppressions for the warnings I see for the local build.

My thought was to avoid suppressing rules that we are not violating (except for those that were documented as slowing down the build)